README.md

```diff
@@ -31,39 +54,32 @@ DAEMON: False
 ```
 Be sure to replace the _SLACK_TOKEN_ in the rtmbot.conf file with the one from the bot you created.
 
-**To get the latest food-bot database from heroku stage:**
-
-First we will need to add you as a collaborator.
+**Be sure to have a postgress database setup named food_bot locally:**
 
 On your terminal,
-
-**Create a database named food_bot locally:**
 ```
 createdb food_bot
 ```
-**Create tables:**
-```
-CREATE TABLE menu_table (id SERIAL PRIMARY KEY,
-                   day VARCHAR(10) NOT NULL,
-                   food VARCHAR(60) NOT NULL,
-                   meal VARCHAR(10) NOT NULL,
-                   option INT NOT NULL,
-                   week INT NOT NULL
-                 );
+You can also use the PgAdmin Tool to do that manually.
 
-CREATE TABLE rating (id SERIAL PRIMARY KEY,
-                    date timestamp without time zone default (now() at time zone 'utc'),
-                    user_id VARCHAR(20),
-                    menu_id INT REFERENCES menu_table(id),
-                    rate INT NOT NULL,
-                    comment TEXT
-                   );
+**Run the following command to create the tables **
+```
+python django_foodbot/manage.py makemigrations
 ```
-**Download the database backup from heroku:**
+```
+python django_foodbot/manage.py migrate
+```
+
+###Get the latest food-bot database from heroku stage.
```

Comment:
@andela-sjames I'm not sure `heroku stage` is easy to understand. Do you mean `foodbot staging app` or something else.